src/components/button-toggle/button-toggle-icon.component.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledButtonToggleIcon } from './button-toggle.style';
+import Icon from '../icon';
+
+function ButtonToggleIcon(props) {
+  if (!props.buttonIcon) return null;
```

Comment:
Could we move this check up a level, into button-toggle.component.js, to conditionally render the component?